kwave/kgrid.py

```diff
@@ -11,25 +11,19 @@
 
 @dataclass
 class kWaveGrid(object):
-    # default CFL number
-    CFL_DEFAULT = 0.3
```

Comment:
I kinda liked this because it shouts "I am constant and you are not supposed to change me" :D Any way we can have the same message with the changes here?